roles/setup_http_store/tasks/main.yml

```diff
@@ -9,7 +9,7 @@
 
     - name: Open http port, zone internal and public, for firewalld
       firewalld:
-        port: "80/tcp"
+        port: "{{ http_port }}/tcp"
```

Comment:
Did you mean to use `http_store_port` here?  I don't see where http_port defined.

Reply:
Its in the default of the role